github/commitstatus_private_test.go

```diff
@@ -1,114 +1,106 @@
 package github
 
-import (
```

Comment:
ah, I had forgotten this file. I think we can just delete it, but I will double-check later.

Reply:
Deleted. These tests are replaced by
- retry/retry_test.go
- github/commitstatus_test.go